README.md

```diff
@@ -1,2 +1,6 @@
 # nlp-timeline-gen
```

Comment:
Maybe change the title to the name of our project (Automatic Character Timeline Generation from Books (NLP))